### PR TITLE
Reduce large repositories update

### LINKED ISSUE #6350: Reducing Large Repositories Doc Update
Re: [Reducing Large Repositories](https://pantheon.io/docs/reducing-large-repos)

Priority: Low

## Issue Description:
Document limitation when using with custom upstream.

## Suggested Resolution
"Existing sites not only need to be repointed but also force-pushed with latest code from upstream. Except if the new upstream is a clone from previous upstream." Then you can use the upstream normally afterwards

### PR DESCRIPTION
**Pull request by wordsmither**
Closes #6350 

## Summary

**[Reducing Large Repositories](https://pantheon.io/docs/reducing-large-repos#prune-and-optimize-large-repositories)** - Clarification re: working with custom upstreams.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/reducing-large-repos.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ You can output the size of your repository by running [`git count-objects -vH`](
 
 ## Prune and Optimize Large Repositories
 
+<Alert title="Note" type="info" >For sites using custom upstream, apply the following steps directly to the custom upstream repository. Force push the updated custom upstream repository branch to the existing sites' Pantheon repository origin.</Alert>
```
Comment (wordsmither) [inline source/content/reducing-large-repos.md:36]: @marfillaster - is the second sentence here HOW they apply the following steps?  Or is it the first step in their process (to be completed before step 1 below)?

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7185](https://pr-7185--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `55.00000000000001` | `57.99999999999999` | `3` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-7185` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/34816/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/34816/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7185--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by wordsmither** [comment]
More comments from Ken:

Having re-read the section, I think it is a bit confusing. Perhaps changing it into just a reminder and placing it at the very end, after step 10?
Note: If the site is using a custom upstream, check also  the custom upstream if it contains large files that can be pruned.
Not really sure if users can infer the steps needed to prune custom upstream which are mostly similar(with some variation) to the steps. (edited)

**Comment by whitneymeredith** [comment]
[Preview](https://pr-7185--pantheon-docs.my.pantheonfrontend.website/docs/reducing-large-repos)

**Comment by whitneymeredith** [comment]
@wordsmither The code example for step 10 includes the git gc and other commands for disk optimization. Can the git prune command be added to the code example instead of adding a note at the end of the doc?

**Comment by wordsmither** [comment]
> Preview

I don't think so.  That step is specific to optimizing disk space.  The note only applies to custom upstream, and is about finding additional large files and then pruning them.

**Comment by whitneymeredith** [comment]
LGTM

**Pull request merged**